Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -483,9 +484,13 @@ public GameObject GetClosestValidGrabbable(Vector3 pointerPosition, Vector3 poin
                         && IsColliderPositionValid(collider, pointerPosition, pointerAxis, queryAngle, queryMinDistance, out colliderHitPoint))
                     {
                         float distance = (pointerPosition - colliderHitPoint).sqrMagnitude;
-                        if (distance < closestDistance)
+                        float volume = collider.bounds.Transform(collider.transform.localToWorldMatrix).Volume();
+
+                        float distanceDiff = distance - closestDistance;
+                        if (distanceDiff < -1.0e-5f || (Mathf.Abs(distanceDiff) < 1.0e-5f && volume < closestVolume))
```

Comment:
Could you create a const float field for the number? Also this line might worth some commenting as well.